src/index.js

```diff
@@ -1,5 +1,12 @@
 import ElementPortal from './ElementPortal';
+import FiberElementPortal from './FiberElementPortal';
+import LegacyElementPortal from './LegacyElementPortal';
 import withElementPortal from './withElementPortal';
 
-export { withElementPortal };
+export {
+  FiberElementPortal,
+  LegacyElementPortal,
```

Comment:
Could you explain why you chose to name-export `FiberElementPortal` and `LegacyElementPortal` here @ngryman? I'm wondering if we really need to expose those.

Reply:
In case someone really needs to pick an implementation over another. As it's a public project I thought it was a good idea as the cost to do so is pretty minimal.

Reply:
I'll remove this.